### PR TITLE
Email search should always convert values to lower case

### PR DESCRIPTION
**Pull request by riteshshrv**
This will make 'email' case insensitive globally
[#133174361]

### CODE REVIEW

**Comment by prakashpp** [comment]
Build is failing

**Comment by prakashpp** [comment]
LGTM

**Comment by sharoonthomas** [comment]
Should explicitly set the new pycountry version

File: setup.py [hunk 1]
```diff
@@ -145,7 +145,7 @@ def run(self):
 # Testing dependencies
 tests_require = [
     'mock',
-    'pycountry',
+    'pycountry==16.11.08',
```
Comment (sharoonthomas) [inline setup.py]: `>=`

File: trytond_nereid/tests/test_auth.py [hunk 2]
```diff
@@ -1201,6 +1201,53 @@ def test_0430_remember_me(self):
...
+
+        self.setup_defaults()
+        app = self.get_app()
+
+        # create a party
+        party1, = Party.create([{'name': 'John Doe'}])
+
+        NereidUser.create([{
+            'party': party1.id,
+            'email': 'john_doe@xyz.com',
```
Comment (sharoonthomas) [inline trytond_nereid/tests/test_auth.py]: make this `JOHN_doe@XyZ.com`

Comment (sharoonthomas) [inline trytond_nereid/tests/test_auth.py]: and then test for login. it won't work.

**Pull request closed without merging**